File_Adapter/CRUD/Read/ReadFileContent.cs

```diff
@@ -155,41 +152,8 @@ public static IEnumerable<object> RetrieveJsonContent(string fileFullPath)
 
         public static bool FromJson(string json, out object result)
         {
-            result = null;
```

Comment:
This removes the support for reading Json Arrays.

Reply:
The FromJson method in the Serialiser Engine already takes care of arrays

Reply:
Thanks. If there's any problem with warnings, then we should just keep the check on first character ('[' or '{') but then call the appropriate method in the Engine (`FromJson` and `FromJsonArray`) instead of replicated the code that is found there. This removes the direct dependency to the Mongo NuGet package and therefore limits the risk to have incompatible versions between toolkits.